account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/Stamper.java

```diff
@@ -156,25 +121,21 @@ public Stamp stamp(String payload) throws GeneralSecurityException {
                 this.bundlePrivateKey);
 
         // Sign with SHA256withECDSA
-        try {
-            Signature signer = Signature.getInstance("SHA256withECDSA");
-            signer.initSign(ecPrivateKey);
-            signer.update(payload.getBytes());
-            byte[] signatureBytes = signer.sign();
+        Signature signer = Signature.getInstance("SHA256withECDSA");
+        signer.initSign(ecPrivateKey);
+        signer.update(payload.getBytes());
+        byte[] signatureBytes = signer.sign();
 
-            // Prepare the stamp structure
-            APIStamp apiStamp = new APIStamp(String.valueOf(this.bundlePublicKey), "SIGNATURE_SCHEME_TK_API_P256",
-                    Hex.encode(signatureBytes));
+        // Prepare the stamp structure
+        APIStamp apiStamp = new APIStamp(String.valueOf(this.bundlePublicKey), "SIGNATURE_SCHEME_TK_API_P256",
```

Comment:
Wait, does this work? `bundlePublicKey` is a `bytes[]`, so isn't `String.valueOf(this.bundlePublicKey)` going to be something like 
`"[B@7344699f"`?